proposals/more-const-types.md

```diff
@@ -0,0 +1,79 @@
+# Unmanaged constant types
+
+* [x] Proposed
+* [ ] Prototype: [Complete](https://github.com/PROTOTYPE_OWNER/roslyn/BRANCH_NAME)
+* [ ] Implementation: [In Progress](https://github.com/dotnet/roslyn/BRANCH_NAME)
+* [ ] Specification: [Not Started](pr/1)
+
+## Summary
+[summary]: #summary
+
+Expand the allowable types for compile-time constants (`const` declarations) to any value type that is fully blittable.
```

Comment:
C# notably doesn't have a concept of `blittable`, only `unmanaged`. `blittable` has a specific meaning, as it pertains to `interop`, and types like `bool` aren't considered `blittable` unless the user opts-in via `DisableRuntimeMarshalling`